src/Tokenizer/Token.php

```diff
@@ -56,6 +56,10 @@ class Token
     public function __construct($token)
     {
         if (\is_array($token)) {
+            if (2 !== \count($token)) {
```

Comment:
:-1:

I believe it's super handy to pass 3-items array as well, eg directly from `token_get_all`, like
`new Token(token_get_all($codeAsAString)[$index]);`

mapping 3-items array to 2 items array, like you need to do later, is not only tedious, but also wasting cpu/ram in `Tokens` class

Reply:
pls change to 2 or 3 elements in array

Reply:
The idea of this PR was to restrict 2 or 3 number to allowing only 2, if we want to allow 2 or 3 the this check is not needed at all.